### DISCUSSION TITLE
Import Functionality Discussion Hub: Let's Focus on More Import Needs!

### DISCUSSION THREAD

**Discussion by Xtdzs** [Xtdzs/Zotero-Pagewise-Import-Tool#3] [Ideas]
Welcome to the Import Functionality Discussion Hub!
﻿
Purpose:
The aim of this platform is to address the pressing need for enhanced import capabilities on various websites, particularly focusing on importing papers into Zotero efficiently. Our primary goal is to develop a method to import all papers from specific search pages into Zotero seamlessly. 
﻿
Objective:
Through collaborative discussion and brainstorming, we seek to identify challenges, propose solutions, and ultimately develop tools or methods that streamline the import process. Our ultimate aim is to enhance the usability and effectiveness of Zotero and similar tools for researchers and academics worldwide.
﻿
Discussion Topics:
﻿
Identifying limitations and challenges faced with current import functionalities, particularly regarding other requirable academic resources.
Sharing experiences and insights on strategies or workarounds used to import papers into Zotero effectively.
Brainstorming innovative solutions and techniques to overcome the limitations of existing import methods.
Collaborating on the development and testing of tools or scripts to automate the import process from specific search pages to Zotero libraries.
Gathering feedback and suggestions from the community to refine and improve the import functionality.

**Comment by aiyou9** [comment]
These websites provide a list of references recommended by ai, hoping to extract the doi from the links.


1、[https://typeset.io/search?q=nano](https://typeset.io/search?q=nano)
![image](https://github.com/Xtdzs/Zotero-Pagewise-Import-Tool/assets/15712822/e1ddfdfc-c3a3-4a24-a1ee-7a46e5e9a21f)

2、[https://elicit.com/](https://elicit.com/)
![image](https://github.com/Xtdzs/Zotero-Pagewise-Import-Tool/assets/15712822/267c931c-6df1-4bbe-97c1-d616bf5a9a9f)

**Reply by Xtdzs** [reply]
Both websites are challenging to scrape. However, I was able to **successfully** scrape the first website because the DOI number is available on the front end. Unfortunately, the second website has the DOI number on the back end, making it **inaccessible**.
I've already uploaded the newest version.

**Reply by aiyou9** [reply]
first website  I use chrome plugin to screen doi link https://chromewebstore.google.com/detail/just-one-page-pdf/fgbhbfdgdlojklkbhdoilkdlomoilbpl/reviews

**Reply by Xtdzs** [reply]
I tried this just now and found it a very useful plugin that can directly filter out all the DOI links from the webpage HTML code! (though it might be a bit cumbersome to copy them one by one, and that maybe the advantage of my tool)
Thank you for recommending this plugin. Its features are comprehensive and useful!
